entrypoint.sh

```diff
@@ -1,4 +1,4 @@
-#/!bin/bash
+#!/bin/bash
 set -m
 electrs&
 websocat -b ws-l:127.0.0.1:1234 tcp:127.0.0.1:50001&
```

Comment:
hardcoding the port may be risky, shall we gather these ports via ENV VARS? We can have defaults if not passed